### PR TITLE
Add tests with docker to CI

### LINKED ISSUE #103: Run some tests in CI
The current CI runs only unit tests.
We want to run some actual tests `lein run test ...` in CI.

### PR DESCRIPTION
**Pull request by yito88**
closes #103

### CODE REVIEW

**Comment by yito88** [comment]
Caching doesn't work well. I'm investigating it.

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        docker-compose up -d
+
+    - name: Cassandra test
+      run: |
+        docker exec jepsen-control bash -c "cd /scalar-jepsen/cassandra && lein run test --workload lwt --nodes n1,n2,n3 --ssh-private-key ~/.ssh/id_rsa"
+
+    - name: ScalarDB test
+      run: |
+        docker exec jepsen-control bash -c "cd /scalar-jepsen/cassandra && lein install"
+        docker exec jepsen-control bash -c "cd /scalar-jepsen/scalardb && lein run test --workload transfer --nodes n1,n2,n3 --ssh-private-key ~/.ssh/id_rsa"
```
Comment (feeblefakie) [inline .github/workflows/test.yml:38]: Can we also add a test for `scalardl`? Or, is there any specific reason for not doing it?

**Review by feeblefakie — COMMENTED** [review]
Thank you!
Left one question for clarification.

File: .github/workflows/test.yml (hunk 1 above)
Comment (yito88) [reply .github/workflows/test.yml:38]: Good point! Because the test requires the binary of ScalarDL server.

**Review by brfrn169 — APPROVED** [review]
LGTM! Thank you!

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**